packages/flutter_tools/templates/create/.travis.yml.tmpl

```diff
@@ -0,0 +1,11 @@
+os:
+  - osx
+osx_image: xcode7.2
+sudo: false
+before_script:
+  - git clone https://github.com/flutter/flutter.git /tmp/flutter
```

Comment:
There are some issues with this approach:

1. This is tracking `master` rather than `alpha`.  Really, the developer should decide on a specific git hash for flutter.git that they update in a controlled manner.  As written, the Travis for a customer will go red whenever we land a change in `master` that breaks their app.

2. Putting the flutter repo in `/tmp/flutter` isn't the pattern we've been recommending.  Instead, we've been recommending that people put flutter in a peer directory to their project.  For example, they'd have a path dependency to `../flutter/packages/flutter`.